src/Tools/GenerateProps/targetgroups.props

```diff
@@ -138,11 +138,11 @@
       <PackageTargetFramework>uap10.1</PackageTargetFramework>
       <NuGetTargetMoniker>UAP,Version=v10.1</NuGetTargetMoniker>
     </TargetGroups>
-    <TargetGroups Include="portable-net45+win8+sl5">
+    <TargetGroups Include="portable_net45+win8+sl5">
```

Comment:
We need it for value tuple, but these strings are purely opaque keys so we can choose whatever value we want.